src/main/java/seedu/exercise/logic/commands/statistic/StatsFactory.java

```diff
@@ -286,6 +292,8 @@ public Statistic getDefaultStatistic() {
         ArrayList<Date> dates = Date.getListOfDates(startDate, endDate);
         ArrayList<String> properties = datesToString(dates);
         ArrayList<Double> values = caloriesByDate(filteredExercise, dates);
-        return new Statistic("calories", LINE_CHART, startDate, endDate, properties, values);
+        double total = getTotal(values);
+        double average = getAverage(total);
+        return new Statistic("calories", LINE_CHART, startDate, endDate, properties, values, total, average);
```

Comment:
can place these `"calories"` and `LINE_CHART` as constant variables with names like `DEFAULT_XXX` in this class.

Reply:
ok